net_handler/net_util.h

```diff
@@ -40,7 +41,6 @@
 typedef enum net_msg {
     RUN_MODE_MSG,
     START_POS_MSG,
-    CHALLENGE_DATA_MSG,
```

Comment:
this change will need to be communicated to all of the other teams, since this means that `LOG_MSG`, `DEVICE_DATA_MSG`, `GAME_STATE_MSG`, etc. are all shifted down by one number. if we don't want to do that / don't want to break backwards compatibility / want to deal with it later, we should leave `CHALLENGE_DATA_MSG` in this enum or replace it with something else, like `NOP_MSG` or something and mark it with a `TODO`

Reply:
I think this is a good time to communicate this with the other teams